game/factions/faction.py

```diff
@@ -575,8 +575,10 @@ def apply_mod_settings(self, mod_settings: Optional[ModSettings] = None) -> None
             self.remove_preset("David's Sling (Semicircle)")
         # swedish military assets pack
         if not mod_settings.swedishmilitaryassetspack:
-            self.remove_vehicle("BV410_RBS70")
-            self.remove_vehicle("BV410_RBS90")
```

Comment:
These seem to have been replaced by RBS_70 & RBS_90, correct? If so, there's some other implications that need to be addressed, such as its usage in the Pretense generator 😅